simapp/amino.go

```diff
@@ -0,0 +1,19 @@
+package simapp
+
+import (
+	"github.com/cosmos/cosmos-sdk/simapp/params"
+	"github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+// AminoJSONTxDecoder returns an Amino JSON TxDecoder using the provided cfg Marshaler
+func AminoJSONTxDecoder(cfg params.EncodingConfig) types.TxDecoder {
```

Comment:
Why not include this in `simapp/params` pkg?

Reply:
I believe we can even delete this file actually now, this was from a previous implementation